### ISSUE TITLE
console.log UI component

### ISSUE THREAD

**Issue by enjalot** [enjalot/building-blocks#86]
People have requested the ability to view console.log output in the app rather than having to open up the console.
This could be in the form of inline-logs in the editor (something we did in Tributary) or perhaps as an overlay on top of the renderer that comes up when a button is clicked or a short-cut key is pressed.

We could use something like this perhaps:
https://github.com/Lapple/react-json-inspector

**Comment by curran** [comment]
Cool idea! JSBin has implemented this feature [http://jsbin.com/](http://jsbin.com/), maybe looking at their implementation will get you closer. Here's a part of it [https://github.com/jsbin/jsbin/blob/master/public/js/runner/proxy-console.js](https://github.com/jsbin/jsbin/blob/master/public/js/runner/proxy-console.js)

**Comment by georules** [comment]
This can be likely accomplished using the technique used to communicate errors from the JS console.  I like the look&feel of the jsbin console.

**Comment by enjalot** [comment]
I agree @georules, the postMessage system would work well. We would also be able to know line numbers.

it looks like the ui implementation isn't all that complicated. someone turning this into a react component would make it fairly straightforward to plugin!
https://github.com/jsbin/jsbin/blob/master/public/js/render/console.js
